src/backend/marsha/core/models/video.py

```diff
@@ -416,6 +416,16 @@ class LiveRegistration(BaseModel):
 
     RESOURCE_NAME = "liveregistrations"
 
+    anonymous_id = models.UUIDField(blank=True, db_index=True, null=True)
+
+    display_username = models.CharField(
```

Comment:
We didn't decided to use `display_name` ?

Reply:
original field is username so I kept the same root

Reply:
Yes but they have not the same purpose.